sw/include/regs/irq_router.h

```diff
@@ -0,0 +1,30 @@
+// Copyright 2023 ETH Zurich and University of Bologna.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+//
+// Robert Balas <balasr@iis.ee.ethz.ch>
+// Alessandro Ottaviano <aottaviano@iis.ee.ethz.ch>
+
+// Header definition for irq_router
+// Note that this is not auto-generated due to limitations of reggen
+
+#ifndef _IRQ_ROUTER_REG_DEFS_
+#define _IRQ_ROUTER_REG_DEFS_
+
+// Register width
+#define IRQ_ROUTER_PARAM_REG_WIDTH 32
+
+// Target selection bitmask control register
+#define IRQ_ROUTER_IRQ_TARGET_MASK_REG_OFFSET(i) (0x0 + 0x4 * i)
+
+#define IRQ_ROUTER_NUM_IRQ_SRCS 59
+#define IRQ_ROUTER_NUM_TARGETS 4
+
+enum car_irq_router_target {
+    IRQ_ROUTER_TARGET_NONE          = 0,
+    IRQ_ROUTER_TARGET_CVA6_0        = 1,
```

Comment:
I think this should be CLIC_CVA6_0 (same below) -> I know that the target is a hart, but for cva6 we made this fully flexible with Nils if you remember, so, for instance, CVA60 as a hart has two interrupt controllers (PLIC and CLIC), which transaltes in two targets from the viewpoint of the irq router